lib/Zonemaster/Backend/DB.pm

```diff
@@ -282,11 +282,7 @@ sub get_test_params {
 
     my $result;
     eval {
-        if ( utf8::is_utf8( $params_json ) ) {
-            $result = decode_json( encode_utf8( $params_json ) );
-        } else {
-            $result = decode_json( $params_json );
-        }
+        $result = decode_json( $params_json );
```

Comment:
> Do you know if this will work for data that was inserted before this PR?

Yes this should not break anything since it is only about using the same encoding when passing (retrieving) data to (from) the drivers. Then data is still stored in UTF-8 in database. Old data won't be altered and will still behave the same.